modules/API/external/external_api.js

```diff
@@ -191,6 +179,34 @@ function parseArguments(args) {
 }
 
 /**
+ * Compute valid values for height and width. If a number is specified it's
+ * treated as pixel units. If the value is expressed in px, em, pt or
+ * percentage, it's used as is.
+ *
+ * @param {any} value - The value to be parsed.
+ * @returns {string|undefined} The parsed value that can be used for setting
+ * sizes through the style property. If invalid value is passed the method
+ * retuns undefined.
+ */
+function parseSizeParam(value) {
```

Comment:
This looks like a private function. Shouldn't we also add the underscore for function if we're doing it for variables.